### ISSUE TITLE
Change 'master' branch to 'main' to be culturally appropriate.

### ISSUE THREAD

**Issue by GeneralTony** [MattW-source/discord-wilfred#306]
If you believe such a change is only trivial I suggest you go read these articles.
https://www.zdnet.com/article/uk-ncsc-to-stop-using-whitelist-and-blacklist-due-to-racial-stereotyping/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/
https://bugs.chromium.org/p/chromium/issues/detail?id=981129#c16

**Comment by MM-coder** [comment]
"Just because something is widely accepted, it doesn't make it correct"

This is a trivial change because you are associating a racial connotation to something that doesn't have any, nor do any of the project maintainers use "master" in reference to slavery.

The articles you posted are regarding the use of whitelist and blacklist, not the use of master in branch names

**Comment by GeneralTony** [comment]
> This is a trivial change because you are associating a racial connotation to something that doesn't have any, nor do any of the 
Are you actually autistic or some sort of white supremist? I suggest you go revise your history to do with slavery and how slaves were controlled by a "master". Use of such terminology in the 21st century is disgusting.

**Comment by FrankKeller** [comment]
For someone trying to be culturally appropriate you do have an interesting choice of insults @GeneralTony 

As someone who's been in the technology industry for well over a decade, I have never looked at the "master" and "slave" terminology used in the computer industry as appropriating slavery. In this context, "master" and "main" mean exactly the same thing, it is the primary branch which all other branches follow. No one is looking at the name of the branch and thinking we should get slaves to write this application for us. No one instinctively associates the name of a branch with some sort of racist culture, unless they **choose** to. 

I believe the change is completely trivial and pointless, the meanings still come across completely the same and the argument that keeping the "master" terminology is perpetuating a racist culture is laughable, name me one person that was turned into a racist sourly by the terminology used in computer science.   

In short, stop choosing to be offended by everything and grow up.

**Comment by MM-coder** [comment]
Is your singular defense calling me a "white supremacist", heck you don't even know my race. 

Master/slave has been used in computer science for decades, it's a well established terminology that has been used for years without any association to slavery, and I'm willing to bet that whomever invited the terminology wasn't thinking about discriminating anyone. 

I'm well revised on my history, thank you, my point here is that **in this specific case** no one is using the term with any racial connotation associated, you are simply attempting to find a problem where it does not exist. 

And also, if you are so against discrimination, why are you calling me autistic, literally discriminating against people who have that mental illness

**Comment by GeneralTony** [comment]
If there's no difference what's so difficult about changing it? By leaving it as it is you're agreeing that slavery should continue to exist in society. Just because something is established for "decades" doesn't mean it can't be changed. Segregation was established for years and was eventually changed for the greater good. You're telling me to grow up but perhaps you should grow up and get out of the 1960s.

**Comment by volcanicer** [comment]
If we could act with a little respect towards each other, that would be greatly appreciated. We don't need people name-calling and flaming each other over an issue like this, thanks.

With regards to changing the name of the branch, it doesn't really matter. While it is more culturally acceptable now to call the branch "main" rather than "master", it doesn't really change anything. It's quite literally a name change. You'd be better off donating to a movement such as Black Lives Matter instead of yelling at people to make a small and trivial change to a rather small GitHub repository.

Thanks for the issue though, and like I said and direct to all parties: please act with respect toward each other.

**Comment by FrankKeller** [comment]
> By leaving it as it is you're agreeing that slavery should continue to exist in society.

Not at all, the branch name isn't exactly called "slavery-is-good" or anything of the likes. In the current context, the term "master" isn't even being used in conjunction with slavery or the term "slave". As my fellow contributor said, you're making a problem out of nothing.

**Comment by GeneralTony** [comment]
So be it. If you guys refuse to change the name of this repo I will have to mark this place as a place of white supremacy and racism. You have 5 minutes.

**Comment by MM-coder** [comment]
If that's what it takes to get you to leave, by all means

**Comment by MM-coder** [comment]
Also since when is it the repository name that's the issue, I thought it was the branch name?

**Comment by FrankKeller** [comment]
Resorting to threats isn't going to achieve anything. Isn't that the sort of thing you woke people are trying to rid the world of, threatening and inappropriate culture?

**Comment by MattW-source** [comment]
The branch name is not being changed for reasons other contributors have already stated.

> Master/slave has been used in computer science for decades, it's a well established terminology that has been used for years without any association to slavery, and I'm willing to bet that whomever invited the terminology wasn't thinking about discriminating anyone.

> With regards to changing the name of the branch, it doesn't really matter. While it is more culturally acceptable now to call the branch "main" rather than "master", it doesn't really change anything. It's quite literally a name change. You'd be better off donating to a movement such as Black Lives Matter instead of yelling at people to make a small and trivial change to a rather small GitHub repository.

> Not at all, the branch name isn't exactly called "slavery-is-good" or anything of the likes. In the current context, the term "master" isn't even being used in conjunction with slavery or the term "slave". As my fellow contributor said, you're making a problem out of nothing.

**Closed and Locked**
